presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -327,6 +327,27 @@ Approximate Aggregate Functions
     for all ``value``\ s. This function is equivalent to the variant of
     :func:`numeric_histogram` that takes a ``weight``, with a per-item weight of ``1``.
 
+.. function:: approx_most_frequent(buckets, value, capacity) -> map<[same as value], bigint>
+
+    Computes the top frequent values up to ``buckets`` elements approximately.
+    Approximate estimation of the function enables us to pick up the frequent
+    values with less memory. Larger ``capacity`` improves the accuracy of
```

Comment:
We may want to add some hint about what are reasonable values for `capacity`, or how it relates to `buckets` (if at all) or to the number of elements processed in total.